content/docs/use-cases/ecommerce.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Ecommerce
+description: Ecommerce page
+---
+
+# Ecommerce
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Dolor pulvinar tincidunt in sed rhoncus lorem. Id eget rhoncus, ipsum mauris nam velit lectus risus. Venenatis non.
+
+<Divider className='w-20'/>
+
+## Ecommerce Platforms
+
+Sed magna nulla pellentesque semper enim.
+
+<div className='grid md:grid-cols-2 gap-8 '>
```

Comment:
Please move this inside the component i mentioned above. The mdx file should contain purely Markdown or components, not jsx.